src/views/CreateArticle.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import ReactQuill from 'react-quill';
+import TagsInput from './TagsInput';
+/**
+ * Create Article view component
+ * @param {object} props Component props
+ * @param {function} props.updateInputHandler updating input values
+ * @param {function} props.updateBodyHandler updating the state of the article body
+ * @param {function} props.imageUploadHandler submit image to cloudinary
+ * @param {function} props.submitHandler submit event handler
+ * @param String props.valueText updated form data body
+ * @param Boolean props.isLoading updated loading state passed as props
+ * @returns {import('@babel/types').JSXAttribute}
+ */
+
+function CreateArticle(props) {
+  const {
+    updateInputHandler,
+    updateBodyHandler,
+    imageUploadHandler,
+    submitHandler,
+    valueText,
+    isLoading,
+  } = props;
+
+  let { responseMessage } = props;
+
+  if (responseMessage === 'title is not a string') {
+    responseMessage = 'title is required';
+  }
+
+  const handleUpdate = (updatedTags) => {
+    props.updateTagHandler(updatedTags);
+  };
+
+  const tagsUpdateHandler = (updatedTags) => {
+    handleUpdate(updatedTags);
+  };
+
+  return (
+    <div>
+      <header className="page-header" data-pg-collapsed>
+        <h3>New Article</h3>
+      </header>
+      <div className="card mb-5">
+        <form className="form p-sm-4 p-3 form-borderless">
+          <div className="form-group mb-4" data-pg-collapsed>
+            <label htmlFor="title" className="font-weight-bold">Title</label>
+            <input type="text" className="form-control text-capitalize" id="title" name="title" onChange={updateInputHandler} required />
+          </div>
+          <div className="form-group mb-4" data-pg-collapsed>
+            <label htmlFor="description" className="font-weight-bold">Short Description</label>
+            <textarea className="form-control textarea-sm" id="description" name="description" onChange={updateInputHandler} required />
+          </div>
+          <div className="form-group mb-4" data-pg-collapsed>
+            <label htmlFor="description" className="font-weight-bold">Details</label>
+            <ReactQuill value={valueText} onChange={updateBodyHandler} />
+          </div>
+          <div className="form-group mb-4" data-pg-collapsed>
+            <label className="font-weight-bold" htmlFor="category">Category</label>
+            <select onChange={updateInputHandler} defaultValue="DEFAULT" className="custom-select" name="categoryId" id="category">
```

Comment:
Great job on this, but I am thinking maybe we shouldn't hard code the category in the options. I think we have a route for it now, we can map through it and have them here.

The reason I am of this opinion is that category might change tomorrow. We would also need to make that change here too. What do you think?

Reply:
@thislekan This is not an extremely pressing issue now. Can it be resolved later? What do you think?

Reply:
@fejiroofficial Alright.